deploy/200-role.yaml

```diff
@@ -45,7 +53,7 @@ rules:
 
 - apiGroups: ['tekton.dev']
   resources: ['taskruns']
-  verbs:     ['get', 'create', 'list', 'watch']
+  verbs:     ['get', 'list', 'watch', 'create', 'delete']
```

Comment:
Small nit: the rest of these `verbs` are sorted alphabetically, except this one and `buildruns` above.